cypress/integration/templates.ts

```diff
@@ -1543,4 +1545,135 @@ context('Template tests', () => {
         .should('have.value', 'Fig_test');
     });
   });
+
+  it('should export template in compatible format', () => {
+    const expectedExport = {
```

Comment:
Maybe it is a better to store this expected result in a separate file under fixtures folder and just read it from there.